tests/test_utils.py

```diff
@@ -22,3 +22,20 @@ def test_build_workspace_path():
     assert build_workspace_path(0) == os.path.join(
         SHARED_VOLUME_PATH, "users/0/workflows"
     )
+
+    assert build_workspace_path(0, workflow_id=1) == os.path.join(
+        SHARED_VOLUME_PATH, "users/0/workflows/1"
+    )
+
+    assert (
+        build_workspace_path(0, workflow_id=1, workspace_root_path="/eos/myanalisis")
+        == "/eos/myanalisis/1"
+    )
+
+    assert build_workspace_path(
+        0, workflow_id=1, workspace_root_path=SHARED_VOLUME_PATH
+    ) == os.path.join(SHARED_VOLUME_PATH, "users/0/workflows/1")
+
+    assert build_workspace_path(
+        0, workflow_id=1, workspace_root_path=SHARED_VOLUME_PATH + "/db"
+    ) == os.path.join(SHARED_VOLUME_PATH, "users/0/workflows/1")
```

Comment:
I think this might be a nice use case of [parametrize](https://docs.pytest.org/en/6.2.x/parametrize.html), as we're calling the same function always.